dev/docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
       #   connections without a password. This is *not* recommended. See
       #   PostgreSQL documentation about "trust"
       POSTGRES_HOST_AUTH_METHOD: trust
+    ports:
+      - "5432:5432"
```

Comment:
thats not supposed to be committed, right?

Reply:
Removed